app/assets/stylesheets/application.css.scss

```diff
@@ -110,6 +110,41 @@ body {
   background-position: center center;
 }
 
+.application-body-map_bree{
+  background-image: url('map_bree.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-map_rivendell{
+  background-image: url('map_rivendell.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-map_minas-tirith{
+  background-image: url('map_minas-tirith.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-map_edora{
+  background-image: url('map_edora.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-map_mordor{
+  background-image: url('map_mordor.png');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space